### PR TITLE
CE-2367 infobox migration message in editor only for the insights list

### PR DESCRIPTION
**Pull request by MariuszCz**
CE-2367 infobox migration message in editor only for the insights list templates

Please merge with config: https://github.com/Wikia/config/pull/1237

### CODE REVIEW

**Comment by lukaszkonieczny** [comment]
lgtm

File: extensions/wikia/PortableInfobox/services/Helpers/PortableInfoboxClassification.php [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+			return false;
+		}
+
+		$titleNeedle = 'infobox';
+		if ( strripos( $titleText, $titleNeedle ) !== false ) {
+			$portableInfoboxNeedle = '<infobox';
+
+			// If a portable infobox markup was found
+			// it means that the template doesn't have a non-portable infobox
+			return strpos( $contentText, $portableInfoboxNeedle ) === false;
```
Comment (dianafa) [inline extensions/wikia/PortableInfobox/services/Helpers/PortableInfoboxClassification.php:31]: It's not very important at all, but I would rephrase this comment to:
`if a portable infobox was not found it means that the template has a non-portable one` because this is what we check here.

File: extensions/wikia/PortableInfobox/services/Helpers/PortableInfoboxClassification.php [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+	 *
+	 * @param string $titleText
+	 * @param string $contentText
+	 * @return bool
+	 */
+	public static function isTitleWithNonportableInfobox( $titleText, $contentText ) {
+		// ignore docs pages
+		if ( stripos( $titleText, '/doc' ) ) {
+			return false;
+		}
```
Comment (dianafa) [inline extensions/wikia/PortableInfobox/services/Helpers/PortableInfoboxClassification.php:23]: What about user pages, sandboxes etc ?

**Comment by dianafa** [comment]
LGTM

File: extensions/wikia/PortableInfobox/services/Helpers/PortableInfoboxClassification.php (hunk 2 above)
Comment (lukaszkonieczny) [inline extensions/wikia/PortableInfobox/services/Helpers/PortableInfoboxClassification.php:23]: I think we should leave it on sandboxes for testing purposes.

**Comment by lukaszkonieczny** [comment]
Thanks @dianafa !
